lib/server.js

```diff
@@ -45,6 +56,8 @@ class DataServer {
 
     this.port = port
     this.wss = null
+
+    if (sqlitePath) sqliteDb.connectDb(sqlitePath)
```

Comment:
I think we should handle possible errors when connecting, and by that also moving this to `open` method instead

Reply:
fixed.